Godeps/Godeps.json

```diff
@@ -1,10 +1,21 @@
 {
 	"ImportPath": "github.com/maruel/panicparse",
-	"GoVersion": "go1.4.2",
+	"GoVersion": "devel +adf4c38 Thu Nov 26 17:42:14 2015 +0000",
```

Comment:
will fix in later

On 11/29/15, M-A <notifications@github.com> wrote:
>> @@ -1,10 +1,21 @@
>>  {
>>  	"ImportPath": "github.com/maruel/panicparse",
>> -	"GoVersion": "go1.4.2",
>> +	"GoVersion": "devel +adf4c38 Thu Nov 26 17:42:14 2015 +0000",
>
> I'd prefer it to be a real go toolset version, e.g. 1.5.1. But I don't know
> why this line is embedded at all, this has no value.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/maruel/panicparse/pull/13/files#r46087035


-- 
- Yasuhiro Matsumoto